addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -426,6 +433,22 @@ def _l10n_it_get_partner_invoice(self, tree, company):
 
         return partner
 
+    def _l10n_it_edi_search_tax_for_import(self, company, percentage, extra_domain=None):
+        extra_domain = extra_domain or []
+        conditions = [
+            ('company_id', '=', company.id),
+            ('amount', '=', percentage),
+            ('amount_type', '=', 'percent'),
+            ('type_tax_use', '=', 'purchase'),
+        ] + extra_domain
+        taxes = self.env['account.tax'].search(conditions).filtered(
+            lambda x: x.invoice_repartition_line_ids.mapped("factor_percent") == [100.0, 100.0])
```

Comment:
Josse, I already changed the name to this function to make it more understandable. In my opinion it's very clear. 
To me, there is no use in putting a comment like "Here we exclude reverse charge taxes".
Why should I ever consider reverse charge taxes while importing invoices?

Reply:
Done anyway 👍🏻  😄

Reply:
Could not we filter out taxes that have a negative factor_percent instead of assuming that every tax will be 100, 100?

lambda t: not any(r.factor_percent < 0 for r in t.invoice_repartition_line_ids)